code/game/machinery/washing_machine.dm

```diff
@@ -204,7 +204,7 @@
 	icon_state = "wm_[state][panel]"
 
 /obj/machinery/washing_machine/attackby(obj/item/W as obj, mob/user as mob, params)
-	/*if(istype(W,/obj/item/screwdriver))
+	/*if(istype(W,/obj/item/screwdriver)) // If anyone ever reenables this, make it a screwdriver_act please
 		panel = !panel
 		to_chat(user, "<span class='notice'>you [panel ? </span>"open" : "close"] the [src]'s maintenance panel")*/
```

Comment:
would you please remove this commented out code.

Reply:
My computer decided to brick itself today. I won't be able to make any changes for the foreseeable future. Do whatever you with this PR, or I can just close it